webapp-django/crashstats/crashstats/jinja2/macros/signature_startup_icons.html

```diff
@@ -1,17 +1,25 @@
+{% macro fission_icon() %}
+  <i class="status-icon fas fa-atom" title="Fission Crash"></i>
+{%- endmacro %}
+
+{% macro potential_fission_icon() %}
+  <i class="status-icon fas fa-atom" title="Potential Fission Crash"></i>
+{%- endmacro %}
```

Comment:
I think I'd rather use fontawesome for all the icons rather than involve multiple 3rd party sets. However, that's not something I wanted to deal with in this pass.

So for now, we've got different kinds of icons and they sort of work together.